tests/assembly/asm/x86-types.rs

```diff
@@ -1,8 +1,10 @@
 //@ revisions: x86_64 i686
 //@ assembly-output: emit-asm
 //@[x86_64] compile-flags: --target x86_64-unknown-linux-gnu
+//@[x86_64] compile-flags: -Zmerge-functions=disabled
 //@[x86_64] needs-llvm-components: x86
 //@[i686] compile-flags: --target i686-unknown-linux-gnu
+//@[i686] compile-flags: -Zmerge-functions=disabled
```

Comment:
Is there any reason to gate with `[...]` in all of these tests, vs. just always passing this flag?

Reply:
I just thought to go with the pattern and add the flags only where needed, what do you think though?

Reply:
Unless I missed something, this is just needless duplication? If you moved this to the non-target-specific flags lower (to where e.g. avx512bw is passed in this test), then it would be clearer that this is just always passed for this test.

Reply:
I have done the changes.